package.json

```diff
@@ -1,54 +1,44 @@
 {
-  "name": "data-protector-use-case",
-  "version": "0.1.0",
+  "name": "vite-project",
   "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```

Comment:
README is outdated, remove everything related to cra and add basic commands for vite

Reply:
Commit: ddc01e703a3fc87e0fcb5b96e03f608eef7104df